MetaMask/metamask-extension
# Add crossfi testnet

## **Description**

Added CrossFi Tesnet chain to the list of chains to support projects logo.



[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/25433?quickstart=1)

## **Manual testing steps**

1. Build mm
2. Add crossfi testnet network
3. Check that crossfi logo appears 

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
